src/web/templates/admin/pairings/pairing_modal.html

```diff
@@ -199,6 +200,88 @@
                     </button>
                 </div>
 
+                {% if experimental_features_enabled %}
```

Comment:
If merged after #976 only (no compile error will raise).
```suggestion
                {% if sharly_chess_config.experimental %}
```